### DISCUSSION TITLE
How i can use queryType with only label?

### DISCUSSION THREAD

**Discussion by b0kky** [grafana/grafonnet#102] [Q&A]
Hello! Could you answer me how i can use queryTypes.withLabelValues(label, metric), without metric? As i see in [doc](https://grafana.github.io/grafonnet/API/dashboard/variable.html#fn-queryquerytypeswithlabelvalues) (fn query.queryTypes.withLabelValues have 2 required fileds: label and metric, but in grafana variables metric may be a option value.
My code is:
```
///
    + var.query.queryTypes.withLabelValues(
      'label',
      '',
///
```
after build via jsonnet become like this:
` "query": "label_values(, label)",`
and Grafana couldn't use it correctly, at variable interface i see this
![изображение](https://github.com/grafana/grafonnet/assets/135844613/5f4d54de-704f-4c32-9d37-7bf561effe98)

**Comment by b0kky** [comment]
Colleagues? may it looks like a bug?

**Comment by Duologic** [comment]
This was indeed a bug, resolved in #108
